doc/nrf/installation/updating.rst

```diff
@@ -16,6 +16,52 @@ You might also want to switch to a newer release or check out the latest state o
 However, if you work with a specific release of the |NCS|, you do not need to update your repositories, because the release will not change.
 For an overview of changes in the latest releases, see :ref:`release_notes`.
 
+.. _using_toolchain_environment:
+
+Using toolchain environment
```

Comment:
I think this needs the phrasing "command-line interface" somewhere. Both in the title and also in the paragraph in line 24. It is not clear enough here that "environment" really means "cli environment"

Reply:
True, let me edit.